### PR TITLE
add motus profile.

### PR DESCRIPTION
**Pull request by jianhong**
Is it possible to add fastq files as input for profiling? Using fasta files as input may not work for mOTUs.

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json
Comment (Midnighter) [inline nextflow_schema.json:421]: I'm not a fan of always using latest master here. We should at least refer to a specific commit hash. Also, is there a reason the script could not be bundled with the MOTUS_DOWNLOADDB module?

**Review by Midnighter — CHANGES_REQUESTED** [review]
Overall the PR looks very good. You committed one file from runtime. I'm sorry for commenting now on how the download DB module works since I didn't help review it but to me this breaks the promise of reproducibility.

**Review by jfy133 — CHANGES_REQUESTED** [review]
Please enter `mOTUs` into `modules.config` and specify the specifc files expected in output.

Please also update `docs/usage.md` to describe what the database path needs to include for mOTUS. For example we need to make sure that it DOES NOT accept (if I understand correctly) the .tar.gz version stored on Zenodo, as the download script does some post-processing, right? So it only accepts a path to a directory (generated BY the download script).

File: nextflow_schema.json
Comment (Midnighter) [reply nextflow_schema.json:421]: Not in `bin` but as within a `templates` directory, this is now supported.

Comment (jfy133) [reply nextflow_schema.json:421]: Oh interesting, I missed this. Any examples?

Comment (jianhong) [reply nextflow_schema.json:421]: The thing is that for different version of mOTUs, the downloadDB are different. By `bin templates`, the script will be fixed, right?

Comment (Midnighter) [reply nextflow_schema.json:421]: Yes, putting the script in a templates directory, it will be fixed. But I actually think that's a good thing and, of course, the mOTUs version is also fixed. So we only have to ensure that the version in the module and the version of the script match.

File: subworkflows/local/db_check.nf
Comment (jfy133) [inline subworkflows/local/db_check.nf:36]: We want to remove that taxprofiler downloads the database for you.

you need to add to the motus CI test an additional commands that downloads and executes the download_db script itself, and then make a special database.csv that just species the path of the download db on the machine (i..e not a remote URL)

Comment (jfy133) [reply subworkflows/local/db_check.nf:36]: No, I mean within the CI tests you have a command to download the  mOTUS download database python script, then run the script, then execute the pipeline

e.g.

```
run: |
      wget <url_to_download_script>
      wget <url_to_motu_specific_database_tsv>.tsv
      python <download_script>.py
      nextflow run nf-core/taxprofiler -profile docker,test_motus  --database <url_to_motu_specific_database_tsv>.tsv
```

File: nextflow_schema.json
Comment (jianhong) [reply nextflow_schema.json]: I removed the download script. So no need to worry about this part in current version.

File: subworkflows/local/db_check.nf
Comment (jianhong) [reply subworkflows/local/db_check.nf]: Great idea!

File: docs/usage.md
Comment (jfy133) [inline docs/usage.md:140]:
```suggestion
  can be thus reused for multiple runs. Users can download the database once using the script above and
```

**Review by jfy133 — APPROVED** [review]
I'm going to do a quick local test, but other than docs phrasing, I think this looks good to me now!

It would be good to have a second step for `motus merge`, but I can add the module to modules and to the pipeline in a separate PR :+1:

**Comment by jfy133** [comment]
@Midnighter if you're also Ok with this now (as you had comments earlier) we can merge this in :)

File: nextflow_schema.json
Comment (jfy133) [reply nextflow_schema.json]: As the database downloading has been removed entirely, I think this point is moot now indeed :+1:

**Comment by jfy133** [comment]
Reviewing @Midnighter 's comments, it looks like the outstanding ones are not irrelevent as that whole functioanlity has been removed.

As I've already reviewed and approved it, I'll let tests pass after merge from upstream and merge.

**Pull request merged**
